package.json

```diff
@@ -28,7 +28,9 @@
     "lint": "eslint ./",
     "lint:fix": "npm run lint -- --fix",
     "format": "prettier --write README.md {lib,docs,tests}/**/*.{js,md}",
-    "test": "jest",
+    "test:local": "jest",
+    "test:ci": "jest --coverage",
+    "test": "is-ci test:ci test:local",
```

Comment:
Curious as to why not simply running `test:ci` on CI? It seems like an unnecessary setup for me... 🤔

Reply:
Mmm it's interesting: on one hand doesn't seem like a necessary dependency as you mentioned, but on the other hand it simplifies the execution of tests and external scripts to just `npm test` without doing something special (we would have to customize travis job to execute `test:ci` rather than default `test` script). So I think is worth the cost of having this new dependency to simplify CI config around tests.

Reply:
If I'm not mistaken, the default Travis CI configuration for Node.js includes a `test` script. I just didn't want to modify the CI config 🙂

Reply:
why have these 2 seperated?  if you want to run w/out coverage you can do `--no-coverage`

Reply:
It's fine, was just curious if there was some other reason 🙂

Reply:
@benmonro I usually prefer having 2 different tests scripts for local and ci just in case there are more things to set up for each (there is another project where I had to set some specific env vars and junit reporter), so it's easier to maintain in that way I think. And even better: it's transparent from executing the script with the package that Thomas added :)